lib/rules/selector-no-vendor-prefix/index.mjs

```diff
@@ -64,10 +64,13 @@ const rule = (primary, secondaryOptions) => {
 				}
 
 				const fix = () => {
+					const index = pseudoNode.sourceIndex;
+					const endIndex = index + pseudoNode.value.length;
```

Comment:
```suggestion
					const endIndex = index + value.length;
```

At this point `value` is the same as `pseudoNode.value`, but maybe that is not intentional?

If the intention is to return the range of the fixed text then these indices should be calculated after the mutation.

Reply:
All the ranges currently return the range of fixed text.
We can discuss to change that behaviour in another PR; this is off topic.

> these indices should be calculated after the mutation

My bad. Fix incoming.